code/modules/antagonists/nukeop/nukeop.dm

```diff
@@ -26,8 +26,9 @@
 	if(!ishuman(owner.current))
 		return
 	var/mob/living/carbon/human/H = owner.current
-
-	H.set_species(/datum/species/human) //Plasamen burn up otherwise, and lizards are vulnerable to asimov AIs
+	
+	if(isplasmaman(H))
+		nukeop_outfit = text2path("[nukeop_outfit]/plasmaman")
```

Comment:
NOOOOOO

Reply:
Have a plasmaman variable instead. Do NOT use text paths, ever.

Reply:
hahahaha oh noooooooo